HW4.py

```diff
@@ -0,0 +1,57 @@
+import pandas as pd
+import plotly.graph_objects as go
+from scipy.stats import ttest_ind
+from sklearn.linear_model import LogisticRegression
+
+
+def dftest():
+    # imported random dataset
+    test = pd.read_csv("test.csv")
+    test["HeartDisease"] = test.HeartDisease.astype("category")
+    test["HeartDisease"] = test["HeartDisease"].map({"Yes": 1, "No": 0})
+    test["Sex"] = test["Sex"].map({"Male": 1, "Female": 0})
+    test["AlcoholDrinking"] = test["AlcoholDrinking"].map({"Yes": 1, "No": 0})
+
+    response_list = [test["HeartDisease"]]
+    predictor_list = [test["AlcoholDrinking"], test["Sex"], test["BMI"]]
+
+    # testing
+    resp_cont_list = []
+    resp_cat_list = []
+    for i in response_list:
+        if i.dtype.name == "category":
+            print(i.name, " is categorical")
+            v1 = i
+            resp_cat_list.append(i)
+        else:
+            print(i.name, " is continuous")
+            resp_cont_list.append(i)
+    pred_cont_list = []
+    pred_bool_list = []
+    for i in predictor_list:
```

Comment:
Good use of for loop to complete major tasks. Remember we also need to do the weighted and unweighted means. Should be easy to implement into your code. Just need to add it to a section of the loop.